### PR TITLE
pyreadline3: new recipe

### PR DESCRIPTION
**Pull request by tschoonj**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/pyreadline3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyreadline3-{{ version }}.tar.gz
+  sha256: ff3b5a1ac0010d0967869f723e687d42cabc7dccf33b14934c92aa5168d260b3
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [not win]
```
Comment (chrisburr) [inline recipes/pyreadline3/meta.yaml:16]: This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

Comment (tschoonj) [reply recipes/pyreadline3/meta.yaml:16]: This one is a Windows only package. You cannot even install it on a different platform.

https://github.com/brgirgis/pyreadline3/blob/46725bb4aee97a7a0dc85cc55ed93a16e22f40d1/setup.py#L19-L21

Comment (chrisburr) [reply recipes/pyreadline3/meta.yaml:16]: Oops, I hastily read it as `skip: True  # [win]` 😅 Sorry about that!

**Comment by tschoonj** [comment]
Thanks!

**Pull request merged**